applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -155,6 +155,15 @@ Button 1:
     * If pressed for more than three seconds, it initiates the factory reset of the device.
       Releasing the button within a 3-second window of the initiation cancels the factory reset procedure.
 
+LED 2:
+   If the :kconfig:option:`CONFIG_BRIDGED_DEVICE_BT` kconfig option is set to ``y`` shows the current state of Bluetooth LE Connectivity Manager state.
+   The following states are possible:
+
+   * Turned Off - The Bridge device is in the idle state and all connections to the Bluetooth LE bridged devices are stable.
```

Comment:
Could we add one additional state? I'm missing here an information that bridge has any device paired. We could split it to:
* Turned Off - The Bridge device is in the idle state and has no Bluetooth LE devices paired.
* Turned On - The Bridge device is in the idle state and all connections to the Bluetooth LE bridged devices are stable.